## Importing necessary libraries

In [1]:
# Libraries to help with reading and manipulating data
import pandas as pd
import numpy as np

# Libraries to help with data visualization
import matplotlib.pyplot as plt
import seaborn as sns
%matplotlib inline 

# Library to help with statistical analysis
import scipy.stats as stats 

## Sampling Distribution

**Q1. Suppose an automobile battery manufacturer claims that the mean lifetime of their battery is 60 months with a standard deviation of 6 months. Suppose the distribution of battery life is approximately normal. Find the probability that the mean lifetime of 40 randomly sampled batteries will be less than 58 months.**

In [6]:
from scipy.stats import norm
mu , sigma = 60, 6
x_bar= 58
s = sigma/np.sqrt(n)
n=40
test_stats = (x_bar-mu)/ (sigma/np.sqrt(n))
test_stats

-2.1081851067789197

In [7]:
np.round(norm.cdf(58, loc = 60, scale = s),4)

0.0175

There is less than 2% chance that the probility of 40 randomly sampled batteries will be less than 58 months.

## Interval Estimation

**Q2. A random sample of 40 households was selected as part of a study on electricity usage, and the number of kilowatt-hours (kWh) was recorded for each household in the sample for the first quarter of 2020. The average usage was found to be 310 kWh. In a very large study in the first quarter of the previous year, it was found that the standard deviation of the usage was 89 kWh.**

**Assuming the standard deviation is unchanged and that the usage is normally distributed, provide an expression for calculating a 95% confidence interval for the mean usage in the first quarter of 2019.**

In [8]:
n= 40 
x_bar = 310
sigma =89
s = sigma / np.sqrt(n)
np.round(norm.interval(0.95, loc= x_bar, scale =s),2)

array([282.42, 337.58])

We are 95% confidence that the mean usage od electricity is between 282.42 and 337.58 kwh

## Hypothesis Testing

**Q3. You are a manager of a Chinese restaurant. You want to determine whether the mean waiting time to place an order has changed in the past month from its previous population mean value of 4.5 minutes. State the null and alternative hypotheses.**

Null Hypothesis: mu0 = 4.5

Alternative Hypothesis: mua !=4.5

**Q4. What is the p-value in a two-tailed z-test for one sample, where the computed test statistic (z-stat) is equal to +2.00?**

In [12]:
from scipy.stats import norm
mu =4.5
z_stat = 2
p_val=1-norm.cdf(2)
p_val

0.02275013194817921

In [16]:
p_value_2_tailed = 2* p_val
print(p_value_2_tailed)
Zstat_2_tailed = z_stat
print(Zstat_2_tailed)

0.04550026389635842
2


**Q5. Samy, Product Manager of K2 Jeans, wants to launch a product line into a new market area. A Survey of a random sample of 400 households in that market showed a mean income per household of 30000 rupees. The standard deviation based on an earlier pilot study of households is 8000 rupees. Samy strongly believes the product line will be adequately profitable only in markets where the mean household income is greater than 29000 rupees. Samy wants our help in deciding whether the product line should be introduced in the new market. Perform statistical analysis and based on that draw a conclusion.**

Assume a level of significance ($\alpha$) of 5%

In [ ]:
H0:  mu = 29000
Ha : mu>29000 

In [17]:
n=400
x_bar, sigma  = 30000 , 8000
mu = 29000
s= sigma/ np.sqrt(n)
test_stat = (x_bar-mu)/ (sigma/np.sqrt(n))
test_stat

2.5

In [19]:
p_val=1-norm.cdf(test_stat)
p_val

0.006209665325776159

Since p value is less than 0.05 so we have enough evidence yto reject the null hypothesis